### ISSUE TITLE
question: bulk source of cpcs for granted non utility patents?

### ISSUE THREAD

**Issue by mustberuss** [USPTO/PatentPublicData#75]
Is there a bulk source of cpcs for granted non utility patents?  The US_Grant_CPC_MCF files  only have cpcs for utility patents.  About half the plant patents in patft have cpc assignments (ex [PP11728](http://patft.uspto.gov/netacgi/nph-Parser?patentnumber=PP11728)) and reissued patents in patft also have cpc assignments (ex [RE46653](http://patft.uspto.gov/netacgi/nph-Parser?patentnumber=RE46653)).

I am a consumer of the patentsview api.  They use the grant master files as their source of cpcs.  As such, they do not return cpcs for plant or reissued patents.

**Comment by bgfeldm** [comment]
I believe Plant and Design Patents are still classified using USPC. And I have not heard of any public plans for them to move to CPC.

**Comment by mustberuss** [comment]
You are right about design patents.  Plant patents are classified using USPCs and about half have also been assigned CPCs in patft.  Try this search in [patft](http://patft.uspto.gov/netahtml/PTO/search-adv.htm) `cpc/A01H5/02 and apt/6` It currently returns 12,335 plant patents with that CPC.  I was looking for a bulk source of them unless that is some sort of mistake.
